app/assets/stylesheets/components/links.scss

```diff
@@ -61,20 +60,26 @@
       background-color: var(--bg-hover);
     }
   }
+
+  &[aria-current='page'] {
+    --bg: var(--link-current-bg);
+    --color: var(--link-current-color);
+    font-weight: var(--fw-bold);
+  }
```

Comment:
In the light theme, we don't have any visual clue here for the current page when it's just an icon in the link (e.g. the notifications page) - at the moment in the live app we have a background on the icon, e.g.

![notification tab with gray background](https://user-images.githubusercontent.com/20773163/145009720-d181f37a-4a33-4c4c-a59b-f9c862d2dc92.png)

but now we have
![notification tab with no background](https://user-images.githubusercontent.com/20773163/145009742-2bb1b72a-4752-4eed-978f-135132be3460.png)

What do you think about adjusting the background in this case?

(In the dark theme, we do have a slightly darker background:
![dark mode](https://user-images.githubusercontent.com/20773163/145013214-38db3604-a0d9-41d3-83cf-fce459ad7709.png))

Reply:
This should now be fixed.